CONTRIBUTING.md

```diff
@@ -90,6 +90,16 @@ We recommend the following commit structure in the following order:
 
 Follow [this guide](https://chris.beams.io/posts/git-commit/) to ensure you're writing good commit messages.
 
+## Creating a Release
+
+To create a production release of mbed-tools, perform the following steps:
+
+1. Check out the `master` branch and ensure it is up to date.
```

Comment:
Not only up to date, but clean as well. We don't want any untracked news files to be added to the changelog on accident, for example.